### PR TITLE
MoveTank turn_degrees convert speed to SpeedValue object

### PR DESCRIPTION
**Pull request by dwalton76**

### CODE REVIEW

File: ev3dev2/motor.py [hunk 1]
```diff
@@ -2102,7 +2109,12 @@ def follow_line(self,
             if sleep_time:
                 time.sleep(sleep_time)
 
-            self.on(left_speed, right_speed)
+            try:
+                self.on(left_speed, right_speed)
+            except SpeedInvalid as e:
+                log.exception(e)
+                self.stop()
+                raise LineFollowErrorTooFast("The robot is moving too fast to follow the line")
```
Comment (WasabiFan) [inline ev3dev2/motor.py:2117]: This seems odd. What if there are other reasons a speed can throw this error?

File: ev3dev2/motor.py [hunk 2]
```diff
@@ -2189,20 +2200,15 @@ def follow_gyro_angle(self,
...
             if sleep_time:
                 time.sleep(sleep_time)
 
-            self.on(left_speed, right_speed)
+            try:
+                self.on(left_speed, right_speed)
+            except SpeedInvalid as e:
+                log.exception(e)
+                self.stop()
+                raise FollowGyroAngleErrorTooFast("The robot is moving too fast to follow the angle")
```
Comment (WasabiFan) [inline ev3dev2/motor.py:2211]: Ditto above

**Review by WasabiFan — COMMENTED** [review]
Generally looks good; I think the semantics of the errors need some work though.

**Comment by WasabiFan** [comment]
I've marked the above threads resolved; I looked again and I don't think it's a significant issue. @dwalton76 Can you confirm that this is OK to merge? If so I will merge, do some testing, and publish a release.

**Comment by dwalton76** [comment]
@wasabifan sorry for being slow to respond. I’ll sync the branch tonight and do some testing then merge.

**Pull request merged**